packages/svelte/src/compiler/phases/3-transform/server/visitors/shared/utils.js

```diff
@@ -270,6 +270,13 @@ export function call_child_renderer(body, async) {
 	return b.stmt(b.call('$$renderer.child', b.arrow([b.id('$$renderer')], body, async)));
 }
 
+/**
+ * @param {BlockStatement | Expression} body
+ */
+export function create_async_block(body) {
```

Comment:
annoys me that this has no parallelism with `call_child_renderer`, and kinda wonder if we should rename that to `create_child` or something

Reply:
renamed to `create_child_block`, for perfect symmetry with `create_async_block`

Reply:
yeah much better